cirq/ops/gate_features.py

```diff
@@ -118,30 +117,6 @@ def default_decompose(
         """
 
 
-class KnownMatrix(metaclass=abc.ABCMeta):
-    """An effect that can be described by a matrix."""
-
-    @abc.abstractmethod
-    def matrix(self) -> np.ndarray:
-        """The unitary matrix of the gate/operation.
-
-        The matrix order is implicit for both gates and operations. For a gate,
-        the matrix must be in order with respect to the list of qubits that the
-        gate is applied to. For an operation, the order must be with respect to
-        its qubits attribute. The qubit-to-amplitude order mapping matches the
-        ordering of numpy.kron(A, B), where A is a qubit earlier in the list
-        than the qubit B.
```

Comment:
It would be good to include this documentation of the qubit ordering and tensor-product convention in `cirq.protocols.unitary` (for gate/op implementers) and maybe also in the user documentation (for users of cirq).

Reply:
Done.